data/src/main/java/io/homeassistant/companion/android/data/url/UrlRepositoryImpl.kt

```diff
@@ -0,0 +1,97 @@
+package io.homeassistant.companion.android.data.url
+
+import io.homeassistant.companion.android.data.LocalStorage
+import io.homeassistant.companion.android.domain.MalformedHttpUrlException
+import io.homeassistant.companion.android.domain.url.UrlRepository
+import java.net.URL
+import javax.inject.Inject
+import javax.inject.Named
+import okhttp3.HttpUrl
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.HttpUrl.Companion.toHttpUrlOrNull
+
+class UrlRepositoryImpl @Inject constructor(
+    @Named("url") private val localStorage: LocalStorage
+) : UrlRepository {
+
+    companion object {
+        private const val PREF_CLOUD_URL = "cloud_url"
```

Comment:
What is `CLOUD_URL` ? There is a Remote UI cloud url and there is a cloudhook (cloud webhook) url.

Reply:
I see now that it's for cloudhook, let's rename it